tf-modules/cumulus/distribution.tf

```diff
@@ -1,4 +1,5 @@
 module "distribution" {
+  count = var.tea_internal_api_endpoint != null ? 1 : 0
```

Comment:
This makes `distribution` conditional and changes its shape. It's now a list so outputs need to reference `module.distribution[x]`